packages/client/src/main.js

```diff
@@ -15,6 +15,7 @@ Vue.use(BootstrapVue);
 // Optionally install the BootstrapVue icon components plugin
 Vue.use(IconsPlugin);
 Vue.use(Vuelidate);
+Vue.use(VueFormulate);
```

Comment:
I think it isn't necessary to mount VueFormulate globally, is it?

Reply:
I tried various variations of this - It seems to only work with mounting it on a per-file basis. It also doesn't play well with exports/imports, or with attempts to register it under `components`... 

Removing it globally and keeping it per-file for now seems to work, I can keep it this way.